docs/release_notes/ibexa_dxp_v4.5.md

```diff
@@ -0,0 +1,215 @@
+
+![React app block](img/4.5_react_app_block.png "React app block")
+
+### Translation comparison
+
+With this release, you can compare different versions of translations of a Content item, including comparison between different languages.
+
+You can now choose between two new options of the view:
+
+- Split - default, side by side view to compare versions of the same or different languages
+- Unified - single column view to compare versions of the same language
+
+Now, when you compare different versions within the same language, the system highlights the changes using colors:
+
+- yellow - content updated
+- blue - content added
+- red - content deleted
+
+![Translation comparison](img/4.5_comparison_view.png "Translation comparison")
+
+For more information, see [Translation comparison](https://doc.ibexa.co/projects/userguide/en/master/content_management/translate_content/#translation-comparison).
+
+### Page Builder for B2B portals [[% include 'snippets/experience_badge.md' %]] [[% include 'snippets/commerce_badge.md' %]]
+
+With this release, you will be able to use Page Builder to create custom Customer Portals for your clients.
+With new Sales rep page block and using all available blocks from the original Page Builder, you can create a unique experience for each customer group.
+Additionally, you can assign each customer group to a specific Customer Portal or create an availability hierarchy based on rules and configuration.
+
+![Page Builder for B2B portals](img/4.5_page_builder_b2b.png "Page Builder for B2B portals")
+
+### Personalization improvements
+
+#### New B2B models in Personalization engine
+
+Personalization engine introduces two new types of models: [last clicked and last purchased B2B, and B2B recurring purchase models](https://doc.ibexa.co/projects/userguide/en/master/personalization/recommendation_models/#b2b-model), dedicated to B2B users.
+Built on the fly, and based on segment groups, the models return actual items clicked by users with the same segment ID and actual bought items.
+B2B recurring purchase model anticipates and predicts purchase of products that were bought recursively within the same segment ID.
+
+### Segment management
+
+Now you can use segmentation logic with operators to build complex segment groups which enable precise filtering.
+With intuitive drag-and-drop interface, define rules, add logic operators and nest segments in segment 
+groups to get the most accurate, precise and targeted recommendations for your customers.
+
+![Segment management](img/4.5_segment_management.png "Segment management logic")
+
+## Other changes
+
+### Customer Data Platform (CDP) configuration
+ 
+In this release, the CDP configuration becomes more generic
+and allows supporting other transport types accepted by CDP.
+Currently, only `stream_file` transport is supported and can be initialized from the configuration.
```

Comment:
Not sure to understand this paragraph. 
"allows supporting other transport types" and "only `stream_file` transport is supported" is kind of the opposite.